lib/pages/main/wallet_contents/send.dart

```diff
@@ -592,6 +594,12 @@ class _SendState extends State<Send> {
       return;
     }
 
```

Comment:
can we display the error message below the field as other validations instead of the snackbar?